src/gt4sd/training_pipelines/tests/test_training_moses_organ.py

```diff
@@ -0,0 +1,81 @@
+"""Moses Organ Trainer unit tests."""
```

Comment:
ORGAN?

Reply:
Apologies if I understand this comment wrong but did you meant that we should change Organ to ORGAN?

Reply:
yep :)

Reply:
I have pushed a change for this, thanks :)